src/bit-components.js

```diff
@@ -153,10 +153,17 @@ export const CameraTool = defineComponent({
 export const MyCameraTool = defineComponent();
 export const MediaLoader = defineComponent({
   src: Types.ui32,
-  flags: Types.ui8
+  flags: Types.ui8,
+  fileId: Types.ui32
 });
 MediaLoader.src[$isStringType] = true;
-export const MediaLoaded = defineComponent();
+MediaLoader.fileId[$isStringType] = true;
+export const MediaLoaded = defineComponent({
+  fileId: Types.ui8,
+  src: Types.ui32
+});
+MediaLoaded.fileId[$isStringType] = true;
+MediaLoaded.src[$isStringType] = true;
```

Comment:
Duplicating these values into this component is workable but non-ideal. Other options:

- Remove `MediaLoaded` component entirely. Add `LOADED` flag to `MediaLoader` and update that flag rather than removing the component on successful load.
- A separate `FileInfo` component which may serve more general purposes in the future. Could be prepared and added by `inflateMediaLoader()`